src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/documents/recallnotification/RecallNotificationContextTest.kt

```diff
@@ -78,12 +78,11 @@ class RecallNotificationContextTest {
   )
 
   private val prisoner = Prisoner(
+    croNumber = prisonerCroNumber,
     firstName = "Bertie",
-    middleNames = "Basset",
```

Comment:
Ditto - Not sure I'd delete this line; it's still valid input - both ways of course.